### DISCUSSION TITLE
全球用戶ig遭停權

### DISCUSSION THREAD

**Discussion by brian431** [1101-GS3073-C/Python-Community-GS3073-C#483] [General]
全球突然出現大量人ig被停權!
不知道這裡有沒有受害者。

https://news.ltn.com.tw/amp/news/life/breakingnews/4107801

**Comment by lx3211** [comment]
目前沒事但很多人開始㊗️祖克柏萬聖節快樂XD

**Comment by Chenszuhong** [comment]
我自己是沒有遇到，但是我朋友就有

**Comment by williamyen0623** [comment]
沒遇到，可能因為我是乖寶寶，從小學開始都有乖乖把所有個資給祖把拔，祖把拔萬歲

**Comment by number32984** [comment]
好險沒把我停權!

**Comment by Ben6uo** [comment]
如果把我的IG停掉 我就跑去滑低卡好了

**Comment by BingRong1217** [comment]
好多大型社交軟體偶爾都會突然出現問題導致大部分人暫時無法使用

**Comment by Yilin20021016** [comment]
阻克博常常這樣了 一旦這樣就是一整天用不了

**Comment by HsuChen88** [comment]
之前我有徵兆時就強迫自己不要使用，但身邊很多人繼續使用都被停權了。

**Reply by skyling0299** [reply]
想問是那些徵兆啊
我以前經營的官方帳號也有無預警被詛過 蠻可惜的
